services/src/modules/plugins/logging.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  ApolloServerPlugin,
+  GraphQLRequestContext,
+  GraphQLRequestContextWillSendResponse,
+} from 'apollo-server-plugin-base';
+import { FastifyInstance } from 'fastify';
+import { Counter, Histogram } from 'prom-client';
+import logger from '../logger';
+
+let errorsInResponseCounter: Counter<string>;
+let requestDurationHistogram: Histogram<string>;
+
+export function createLoggingPlugin(fastifyInstance: FastifyInstance): ApolloServerPlugin {
+  return {
+    serverWillStart() {
+      const promClient = fastifyInstance.metrics?.client;
+      errorsInResponseCounter = new promClient.Counter({
```

Comment:
Maybe we can do `graphql_result{status='success|error'}` or something like that instead?